docs/source/_json/types.resp

```diff
@@ -3,63 +3,68 @@ Content-Type: application/json
 
 [
   {
-    "@id": "http://localhost:55001/plone/@types/Collection", 
```

Comment:
@tisto It was an .editorconfig in the plone-react root, as I develop for that currently.
I've added an .editorconfig here that prevents this behaviour for .resp files.